core/src/main/scala/org/apache/spark/metrics/source/AccumulatorSource.scala

```diff
@@ -64,10 +64,11 @@ class DoubleAccumulatorSource extends AccumulatorSource
  */
 @Experimental
 object LongAccumulatorSource {
-  def register(sc: SparkContext, accumulators: Map[String, LongAccumulator]): Unit = {
+  def register(sc: SparkContext, accumulators: Map[String, LongAccumulator]): Source = {
     val source = new LongAccumulatorSource
     source.register(accumulators)
     sc.env.metricsSystem.registerSource(source)
+    source
```

Comment:
What is the reason of changing this public API and exposing the package private `Source`?